app/templates/output.html

```diff
@@ -1,17 +1,23 @@
 {% include 'header.html' without context %}
 <div class="container">
-    <div class="row">
-      <div class="col">
-          <div class="card">
-              <div class="card-body">
-                <h5>Encrypted secret</h5>
-                <div id="sealed-secret-content"><pre>{{ sealedSecret }}</pre></div>
-                <button id="copy-button" type="button" class="btn btn-link" onclick="CopyToClipboard('sealed-secret-content')"><i class="material-icons">insert_drive_file</i> copy secret to clipboard</button>
-              </div>
-          </div>            
-        <form action="/">
-            <input class="mt-4 btn btn-light btn-lg btn-block" type="submit" value="Encrypt another secret">
-        </form>
+  <a href="/">
+    <p style="text-align:center;"><img src="static/kubeseal-webgui-logo.jpg"
```

Comment:
Style in die CSS-Datei.

Reply:
Ggf. sollten wir auch einfach das Banner in den Header einbauen, dann müssen wir das nicht zweimal deklarieren.

Reply:
Guter Punkt